pkg/controllers/management/agentupgrade/controller.go

```diff
@@ -40,20 +44,24 @@ func Register(ctx context.Context, context *config.ManagementContext) {
 	context.Apps.DaemonSets("").Controller().AddHandler(ctx, "agent-upgrade", h.OnDaemonSetChange)
 }
 
+// OnDeploymentChange checks if it should delete a namespaced deployment and if so, deletes it.
```

Comment:
Good idea! Based on the godocs, https://pkg.go.dev/github.com/fluhus/godoc-tricks#hdr-Documentation_Links [shouldDelete] should work in the comment but my local godoc server doesn't provide a reference for that. It will for package paths though. Not sure if this is an issue due to it being a local server or something

Reply:
Oh okay. I know that the IDE will provide a pointer when comments are written that way, which is useful when jumping around the code in Goland

Reply:
Yeah. I'll leave that reference as `shouldDelete` so it'll at least help devs auto jump to it in the code.